lua/lvim/core/autopairs.lua

```diff
@@ -46,6 +46,10 @@ function M.config()
   }
 end
 
+local function on_confirm_done(...)
```

Comment:
this function needs to be outside of the setup function, otherwise it changes its address with each reload